omnilib/control.lua

```diff
@@ -1,408 +1,343 @@
 require("controlFunctions")
+--
+    --
+        --
+            -- 
+                --
+                    --
+                        --
+                            --
+                                --
+                                    -- RIP MT. OMNISSIAH
+                                --
+                            --
+                        --
+                    --
+                --
+            --
+        --
+    --
+--
```

Comment:
To be clear, this was requested by Zelos.

Reply:
As it should be, leave some comedy in the code :D